gateway/src/cluster/impl.rs

```diff
@@ -316,6 +316,34 @@ impl Cluster {
             .map_err(|source| ClusterCommandError::Sending { source })
     }
 
+    /// Send a raw command to the specified shard.
+    ///
+    /// # Errors
+    ///
+    /// Returns [`ClusterCommandError::Sending`] if the shard exists, but
+    /// sending it failed.
+    ///
+    /// Returns [`ClusterCommandError::ShardNonexistent`] if the provided shard
+    /// ID does not exist in the cluster.
+    ///
+    /// [`ClusterCommandError::Sending`]: enum.ClusterCommandError.html#variant.Sending
+    /// [`ClusterCommandError::ShardNonexistent`]: enum.ClusterCommandError.html#variant.ShardNonexistent
+    pub async fn raw_command(&self, id: u64, value: String) -> Result<(), ClusterCommandError> {
```

Comment:
This would probably be better as a vec of bytes, I think that would match the usual rust-y idiomatic "raw" descriptor more. Thoughts, anyone?

Reply:
`command_raw` is more of a namespace-y name